### PR TITLE
Cargo update mk2

### PR DESCRIPTION
**Pull request by Rayfod**
## About The Pull Request

- Some cargo changes were added. Including some signs that should help with who does what with cargo. - Baron approved.
- FEV changed to 9000, flat out.
- Formal clothes
- Binos crate
- Stimpak cost changes
- Two books added (Codex and Chain that binds)
- Commander headset for people that are missing it
- Price changes
- Removal of the field generator, since it can be used to completely block the entrance

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/modules/cargo/exports/large_objects.dm [hunk 1]
```diff
@@ -88,20 +88,10 @@
...
+	cost = 9000
+	unit_name = "FEV solution"
 	export_types = list(/obj/item/reagent_containers/glass/bottle/FEV_solution)
 
-/datum/export/large/fev/FEV_solution_two
-	cost = 10000
-	unit_name = "FEV mk2"
-	export_types = list(/obj/item/reagent_containers/glass/bottle/FEV_solution/two)
-
-/datum/export/large/fev/FEV_solution_curling
```
Comment (ProbablyCarl) [inline code/modules/cargo/exports/large_objects.dm:100]: Significantly more dangerous. No incentive to ship it?

File: code/modules/cargo/packs/security.dm [hunk 2]
```diff
@@ -52,6 +52,14 @@
 					/obj/effect/spawner/lootdrop/f13/attachments,
 					/obj/effect/spawner/lootdrop/f13/gunparts/tier3)
 
+/datum/supply_pack/security/binocularsf13
```
Comment (ProbablyCarl) [inline code/modules/cargo/packs/security.dm:55]: Maybe adjust the name and include map tools, as well?

File: code/modules/cargo/exports/large_objects.dm (hunk 1 above)
Comment (Rayfod) [reply code/modules/cargo/exports/large_objects.dm:100]: Sure there is, the code did not work previously. It was stuck on only giving 10000 caps, which was a bit overturned. And all the FEV was shipped in the shuttle.
9000 works a lot better when the balance of the increased material export gets taken into consideration.

File: code/modules/cargo/packs/security.dm (hunk 2 above)
Comment (Rayfod) [reply code/modules/cargo/packs/security.dm:55]: I guess I can add some mortal shells as well, then.

**Pull request merged**